ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/validation/CustomValidators.java

```diff
@@ -78,7 +79,7 @@
         throw new KsqlRestException(Errors.queryEndpoint(statement.getStatementText()));
       }),
   PRINT_TOPIC(PrintTopic.class, PrintTopicValidator::validate),
-
+  ALTER_SYSTEM_PROPERTY(AlterSystemProperty.class, StatementValidator.NO_VALIDATION),
```

Comment:
this should not be NO_VALIDATION - you should just not set anything for AlterSystemProperty and the validator should then use the ValidatedCommandFactory to validate `ALTER SYNTAX`

Reply:
When I remove this line I get:

```
Do not know how to validate statement of type: class io.confluent.ksql.parser.tree.AlterSystemProperty
Known types: [class io.confluent.ksql.parser.tree.ListConnectorPlugins, class io.confluent.ksql.parser.tree.ListTables, class io.confluent.ksql.parser.tree.ListTopics, class io.confluent.ksql.parser.tree.Query, class io.confluent.ksql.parser.tree.DropConnector, class io.confluent.ksql.parser.tree.ListQueries, class io.confluent.ksql.parser.tree.DescribeStreams, class io.confluent.ksql.parser.tree.UndefineVariable, class io.confluent.ksql.parser.tree.DefineVariable, class io.confluent.ksql.parser.tree.CreateConnector, class io.confluent.ksql.parser.tree.PrintTopic, class io.confluent.ksql.parser.tree.ShowColumns, class io.confluent.ksql.parser.tree.DescribeConnector, class io.confluent.ksql.parser.tree.ListConnectors, class io.confluent.ksql.parser.tree.ListFunctions, class io.confluent.ksql.parser.tree.ListProperties, class io.confluent.ksql.parser.tree.SetProperty, class io.confluent.ksql.parser.tree.Explain, class io.confluent.ksql.parser.tree.DescribeFunction, class io.confluent.ksql.parser.tree.UnsetProperty, class io.confluent.ksql.parser.tree.ListStreams, class io.confluent.ksql.parser.tree.ListVariables, class io.confluent.ksql.parser.tree.InsertValues, class io.confluent.ksql.parser.tree.ListTypes, class io.confluent.ksql.parser.tree.DescribeTables]
```
When I do keep this line it still uses the `ValidatedCommandFactory` to validate `ALTER SYNTAX`. Is the CustomValidators specification just extra maybe?